.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,64 @@
+name: Bug Report
+description: Create a report to help us improve
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        Please make sure to fill out the entire form below,
```

Comment:
Please use longer lines, it looks like poetry when rendered :)

Reply:
Sure. I just copied it from other projects. Let me fix it.